src/main/java/core/basesyntax/square.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class square extends Figural {
```

Comment:
Названия классов мы пишем с большой буквы
```suggestion
public class Square extends Figural {
```